oshell.sh

```diff
@@ -126,14 +182,15 @@ function oci_set_tenancy() {
   fi
 
   echo ""
-  oshiv config
+  oci_config_print
```

Comment:
The purpose of running `oshiv config` here was to validate the tenancy and compartment against the OCI API. I wonder if we could do that relatively easy using the OCI CLI? 

I found this validation step to be pretty helpful as I have often passed in invalid tenancies /compartments.